lua/obsidian/ui.lua

```diff
@@ -515,14 +691,30 @@ local function update_extmarks(bufnr, ns_id, ui_opts)
 
     -- Check if inside a code block or at code block boundary. If not, update marks.
     if string.match(line, "^%s*```[^`]*$") then
-      inside_code_block = not inside_code_block
+      inside_code_block = true
```

Comment:
Why this change?

Reply:
Oh whoops, I didn't mean to add this. Let me revert.